includes/class-admin.php

```diff
@@ -186,4 +191,12 @@ public static function admin_init() {
 	 */
 	public static function enqueue_scripts() {
 	}
+
+	/**
+	 * Is updating the Node settings from URL?
+	 */
+	public static function is_updating_from_url() {
```

Comment:
Can this live in the class that will actually process the request instead?

Reply:
Done